server/core/src/irods_api_test_harness.cpp

```diff
@@ -0,0 +1,16 @@
+#include "rodsClient.h"
+#include "connection_pool.hpp"
+#include "experimental_plugin_framework.hpp"
+#include "irods_client_api_table.hpp"
+
+int main(int argc, char* argv[])
+{
+    if(argc < 3) { return 1; }
+
+    load_client_api_plugins();
+
+    std::atomic_bool f{};
+    std::cout << irods::experimental::api::client{}(
+                     irods::make_connection_pool()->get_connection(), f,
```

Comment:
does this work?  the connection pool would go out of scope before the connection could be used, right?

Reply:
Seems to be working fine?